libs/platform/src/lib/components/dynamic-page/dynamic-page.component.scss

```diff
@@ -4,13 +4,18 @@
     height: 3rem;
 }
 
-.tab-sticker {
-    left: 0;
-    right: 0;
-    position: relative;
+.header-sticker, .tab-sticker {
+    position: absolute;
+    width: 100%;
     top: 0;
 }
```

Comment:
Could you keep it static instead of absolute?

Reply:
changed to static as suggested